contracts/IRecovery.sol

```diff
@@ -38,17 +38,17 @@ contract IRecovery {
   function isInRecoveryMode() public view returns (bool inRecoveryMode);
 
   /// @notice Set new colony recovery role.
-  /// Can be called by founder.
```

Comment:
As a bit of a cleanup and terminology consistency, can you replace all "founder" instances in tests with "root". My search found 36 instances which all looked straight forward search/replace.

Reply:
Yes, but the one distinction I'd make between a "founder" as the account which creates the colony vs. the "founder" role. I think it's fine to have an account called "founder", which may have multiple roles (including `root`). But I've updated the references to a `founder` _role_ to refer to the `root` role.

Reply:
Do we have a "founder" role still?

Reply:
Nope, which is why I've removed the references to a `founder` _role_.

Reply:
However, I have _re-implemented_ them in terms of the other roles for the Glider release, for the reason that the Dapp has not finished designing for the new roles. Those functions are deprecated though and we should strive to remove them as quickly as possible.